### PR TITLE
[1.8.x] Fixed #25717 -- Changed local name to Slovak language

### PR DESCRIPTION
**Pull request by andersonresende**
https://code.djangoproject.com/ticket/25717

### CODE REVIEW

**Comment by timgraham** [comment]
@claudep, how does this affect translations? Is it safe to backport?

**Comment by claudep** [comment]
We shouldn't touch the po file, this will be handled through Transifex by Slovak translators. The 'name_local' change won't affect translations at all.

File: django/conf/locale/__init__.py [hunk 1]
```diff
@@ -417,7 +417,7 @@
         'bidi': False,
         'code': 'sk',
         'name': 'Slovak',
-        'name_local': 'slovenský',
+        'name_local': 'slovensky',
```
Comment (claudep) [inline django/conf/locale/__init__.py]: Didn't the OP ask for capitalization, too?

**Comment by claudep** [comment]
Fixed in b0ad5081d5

**Pull request closed without merging**